sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -1,95 +1,45 @@
-    (
-      -- The first check is for Fx 103+, the last two checks are for Fx 102 and prior.
-      improve_suggest_experience_checked
-      OR request_id IS NOT NULL
-      OR scenario = 'online'
-    ) AS suggest_data_sharing_enabled,
+    (metrics.boolean.quick_suggest_improve_suggest_experience) AS suggest_data_sharing_enabled,
   FROM
-    contextual_services.quicksuggest_impression
+    `moz-fx-data-shared-prod`.firefox_desktop.quick_suggest
+  WHERE
+    metrics.string.quick_suggest_ping_type IN ("quicksuggest-click", "quicksuggest-impression")
   UNION ALL
   SELECT
-    context_id,
-    DATE(submission_timestamp) AS submission_date,
-    'suggest' AS source,
-    'click' AS event_type,
-    'desktop' AS form_factor,
-    normalized_country_code AS country,
-    metadata.geo.subdivision1 AS subdivision1,
-    advertiser,
-    release_channel,
-    position,
-    CASE
-      WHEN request_id IS NULL
-        THEN 'remote settings'
-      ELSE 'merino'
-    END AS provider,
-    match_type,
-    SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
-    (
-      -- The first check is for Fx 103+, the last two checks are for Fx 102 and prior.
-      improve_suggest_experience_checked
-      OR request_id IS NOT NULL
-      OR scenario = 'online'
-    ) AS suggest_data_sharing_enabled,
-  FROM
-    contextual_services.quicksuggest_click
-  UNION ALL
-  SELECT
-    context_id,
-    DATE(submission_timestamp) AS submission_date,
-    'topsites' AS source,
-    'impression' AS event_type,
-    'desktop' AS form_factor,
-    normalized_country_code AS country,
-    metadata.geo.subdivision1 AS subdivision1,
-    advertiser,
-    release_channel,
-    position,
-    CASE
-      WHEN reporting_url IS NULL
-        THEN 'remote settings'
-      ELSE 'contile'
-    END AS provider,
-    -- `match_type` is only available for `quicksuggest_*` tables
-    NULL AS match_type,
-    SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
-    -- 'suggest_data_sharing_enabled' is only available for `quicksuggest_*` tables
-    NULL AS suggest_data_sharing_enabled,
-  FROM
-    contextual_services.topsites_impression
-  UNION ALL
-  SELECT
-    context_id,
+    metrics.uuid.top_sites_context_id AS context_id,
     DATE(submission_timestamp) AS submission_date,
     'topsites' AS source,
-    'click' AS event_type,
+    IF(metrics.string.top_sites_ping_type = "topsites-click", "click", "impression") AS event_type,
     'desktop' AS form_factor,
     normalized_country_code AS country,
     metadata.geo.subdivision1 AS subdivision1,
-    advertiser,
-    release_channel,
-    position,
+    metrics.string.top_sites_advertiser AS advertiser,
+    client_info.app_channel AS release_channel,
+    metrics.quantity.top_sites_position AS position,
     CASE
-      WHEN reporting_url IS NULL
+      WHEN NULLIF(metrics.string.top_sites_request_id, "") IS NULL
         THEN 'remote settings'
       ELSE 'contile'
     END AS provider,
```

Comment:
@jmsilverman and here (may be different because it's top sites here, and quick suggest in the other two places)

Reply:
ignore this one, it's a typo